Formula/code-server.rb

```diff
@@ -0,0 +1,25 @@
+require "language/node"
+
```

Comment:
```suggestion
require "language/node"

```

Doesn't look like you're using this any more.